HISTORY.md

```diff
@@ -1,3 +1,7 @@
+# v2.3.0
```

Comment:
Version changes should be part of the release process rather than the PR. Could this change please be removed.